Metagenomics/tutorials/mothur-miseq-sop.md

```diff
@@ -0,0 +1,1516 @@
+```
+
+These data tell us that OTUs 1, 2, and 3 was significantly different between the early and late samples.
+
+> ### :question: Question
+>
+>  Which of the top 10 OTUs in your output were significantly different beween early and late samples?
+>
+> <details>
+>  <summary> Click to view answer</summary>
+>  Looking at the p-value cut-off and using your favorite cutoff threshold (say 0.01).
+>  Answer to the question is all OTUs with a value lower than this threshold. Note that these OTU labels may
+>  be different for you and may very between one repetition of this tutorial to the next, and therefore may
+>  vary between you and your neighbour as well.
+> </details>
+{: .question}
+
+Another non-parametric tool we can use as an alternative to metastats is lefse:
+
+> ### :pencil2: Hands-on: Lefse
+>
+> - **Tool:** Lefse
+> - **Parameters:**
+>   - Set **shared** to Subsample.shared
+>   - Set **design** to `mouse.time.design`
+{: .hands_on}
+
+Looking at the top of the lefse summary file we see:
+
+```
+OTU        LogMaxMean  Class   LDA         pValue
+Otu0001    4.41671     Late    3.91585    0.000601825
+Otu0002    4.86254     Late    4.20329    0.000695271
+Otu0003    4.18358     Late    3.82749    0.00022674
+Otu0004    4.4691      -
+Otu0005    4.84546     -
+```
+
+Again, OTUs 1, 2, and 3 are significantly different between the two groups and are significantly elevated in the late samples
+
+Finally, Mothur has an implementation of the random forest algorithm build into her as classify.rf. This will tell us which features (i.e. OTUs) are useful in discriminating between the two groups of samples:
+
+> ### :pencil2: Hands-on: Classify.rf
+>
+> - **Tool:** Classify.rf
+> - **Parameters:**
+>   - Set **shared** to Subsample.shared
+>   - Set **design** to `mouse.time.design`
+{: .hands_on}
+
+in the logfile we see:
+
+```
+Creating 100 (th) Decision tree
+numCorrect = 19
+forrestErrorRate = 0
+confusion matrix:
+        Early    Late    time
+Early   9        0       0
+Late    0        10      0
+time    0        0       0
+```
+
+We can ignore the time row and column and see that our samples were all correctly assigned to the proper groups. Looking at `summary` output, we see the top 10 OTUs that resulted in the greatest mean decrease in activity were:
+
+```
+OTU        Mean decrease accuracy
+Otu0038    0.21
+Otu0003    0.15
+Otu0091    0.14
+Otu0096    0.13
+Otu0024    0.12
+Otu0006    0.1
+Otu0011    0.1
+Otu0015    0.09
+Otu0082    0.08
+Otu0042    0.07
+```
+
+# Step 4: Visualisations
+
+Mothur does not have a lot of visualisation tools built in, but external tools may be used for this. For
+instance we can convert our shared file to the more widely used `biom` format and view it in a platform like
+[Phinch](http://www.phinch.org/).
+
+## Phinch
+
+> ### :pencil2: Hands-on: Phinch
+>
+> - **Tool:** Make.biom
+> - **Parameters:**
+>   - Set **shared** to Subsample.shared
+>   - Set **constaxonomy*
```

Comment:
It could be nice to have a scheme at the end to sum up the different steps. What do you think?